src/charts/SalesChannels.js

```diff
@@ -15,7 +14,7 @@ export default function SalesChannel(selectedYear){
 
   useEffect(()=>{
     const accessToken=getAccessToken();
-    api.get(`http://127.0.0.1:8000/projectcreation-count/${selectedYear.selectedYear}/`, {
+    api.get(`http://127.0.0.1:8000/api/projectcreation-count/${selectedYear.selectedYear}/`, {
```

Comment:
Also, the authorization header is unnecessary here. You have interceptors in Api.js file.

// Following is related to the Home.js file, but it can be applied throughout the entire project.

`
     useEffect(() => {
    const getData = async () => {
      try {
        const [statistics, hours] = await Promise.all([
          api.get(`/project-statistics/${selectedYear}/`),
          api.get(`/project-hours/${selectedYear}/`),
        ]);
        setData(statistics.data);
        setProjectHours(hours.data);
      } catch (error) {
        console.error(error);
      }
    };

    getData();
  }, [selectedYear]);